.github/workflows/test-application.yaml

```diff
@@ -11,7 +11,7 @@ on:
 
 jobs:
     test:
-        name: 'PHP ${{ matrix.php-version }} ${{ matrix.dependencies}}'
+        name: "PHP ${{ matrix.php-version }} ${{ matrix.dependencies }} ${{ matrix.dev-dependencies && 'dev' }}"
```

Comment:
@jordisala1991 i added the dev-dependencies flag to the build name so it is distinct.